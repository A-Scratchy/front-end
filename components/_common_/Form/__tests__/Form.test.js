/* eslint-env jest */
import React from 'react';
import createSnapshotTest from 'test-utils/createSnapshotTest';

import Form from 'components/_common_/Form/Form';

describe('Form', () => {
  it('should render with required props', () => {
    createSnapshotTest(
      <Form>
        <p>Test</p>
      </Form>,
    );
  });

  it('should render with many props assigned', () => {
    createSnapshotTest(
      <Form className="login-form">
        <p>Test</p>
      </Form>,
    );
  });
});
