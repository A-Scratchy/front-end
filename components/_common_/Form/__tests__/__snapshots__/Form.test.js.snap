// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form should render with all props assigned 1`] = `
<form
  className="form login-form"
>
  <p>
    Test
  </p>
</form>
`;

exports[`Form should render with required props 1`] = `
<form
  className="form"
>
  <p>
    Test
  </p>
</form>
`;
